README.md

```diff
@@ -19,9 +19,9 @@
     - [Community, discussion, contribution, and support](#community-discussion-contribution-and-support)
```

Comment:
Want to clean up `#top-level-title` and `#what-is-the-xxx` while you're in here?